packages/pds/src/app-view/db/tables/feed-item.ts

```diff
@@ -0,0 +1,13 @@
+export const tableName = 'feed_item'
+
+export interface FeedItem {
+  uri: string
+  cid: string
+  type: 'post' | 'repost'
+  postUri: string
+  postAuthorDid: string
```

Comment:
I think you could drop this column since - like replyRoot/replyParent - you can grab this off the join 🤔

Reply:
One reason to keep it would be if we want to put any indexes on it for building feeds, but I think where things landed it didn't require any, so seems all good to remove 👍